forge-game/src/main/java/forge/game/staticability/StaticAbilityContinuous.java

```diff
@@ -631,6 +633,15 @@ public static CardCollectionView applyContinuousAbility(final StaticAbility stAb
                         affectedCard.addNewPTByText(state.getBasePower(), state.getBaseToughness(), se.getTimestamp(), stAb.getId());
                     }
                 }
+                if (stAb.hasParam("Incorporate")) {
+                    final ManaCost manaCost = new ManaCost(new ManaCostParser(stAb.getParam("Incorporate")));
```

Comment:
the hidden statics for it aren't there yet - or you want to leave it unconnected for now...?